cmake/compile_definitions/linux.cmake

```diff
@@ -234,14 +234,22 @@ if (${SUNSHINE_TRAY} EQUAL 0 AND SUNSHINE_REQUIRE_TRAY)
     message(FATAL_ERROR "Tray icon is required")
 endif()
 
+option(SUNSHINE_USE_INPUTTINO "Use inputtino for virtual input" ON)
```

Comment:
If we're going to use an option, the option flag should go into https://github.com/LizardByte/Sunshine/blob/8316f44e10f3f6ef27fc27b3ef6e69f359b1b667/cmake/prep/options.cmake#L28

But I don't see much benefit it making this optional. I did see your comment:

> I've kept the original input.cpp so that we can easily compare the two implementations and possibly add what's missing

and

> the idea would be to completely replace that implementation with this

I'd like to get to that point before merging this, if we can get enough people testing.

Or, if we can't get enough testers, we could rename the option to `SUNSHINE_USE_LEGACY_INPUT` (OFF by default) which would clearly indicate that it's not maintained.

Reply:
Thanks for pointing to `options.cmake` I must have missed it, makes sense.